app/decorators/physical_switch_decorator.rb

```diff
@@ -1,6 +1,6 @@
 class PhysicalSwitchDecorator < MiqDecorator
   def self.fonticon
-    'ff ff-network-switch'
+    'pficon ff ff-network-switch'
```

Comment:
The `pficon` is probably wrong, you would need something like `fa-2x` or `fa-4x` depending on what size do you desire. Also if you change this globally, it can break other stuff. I'd suggest appending the class elsewhere, ideally right where it gets rendered.

Reply:
Tried adding, fa-2x, fa-3x, fa-Xx, it make it bigger, but still above the number.

![screenshot-localhost-3000-2018 09 03-09-41-03](https://user-images.githubusercontent.com/3654285/44987822-f1019700-af5e-11e8-93ac-aed9bf9772ca.png)

Reply:
Okay, I'm not that good with CSS :disappointed: but you definitely don't want to make this global in the decorator.